inc/class-s3-uploads.php

```diff
@@ -523,4 +523,43 @@ public function set_attachment_private_on_generate_attachment_metadata( array $m
 
 		return $metadata;
 	}
+
+	/**
+	 * Update the metadata headers on an attachment's metadata by copying it over itself.
+	 *
+	 * @param integer $attachment_id
+	 * @param string $values associative array of metadata. Existing values for a key will be overwritten.
+	 * @return void
+	 */
+	public function set_attachment_meta( int $attachment_id, array $values ): ?WP_Error {
```

Comment:
Types in php give me the shivers.

Reply:
I'm using the same signature as the other set_attachment function